src/ape_plugins/_cli.py

```diff
@@ -21,6 +21,17 @@ def cli():
     Command-line helper for managing installed plugins.
     """
 
+def _remove_prefix_add_version(set_convert,_prefix, version = None):
+    set_print= set()
+    for i in set_convert:
+        if i.startswith(_prefix):
+            text = i[len(_prefix):]
+        if version:
+            set_print.add(f"{text}    ({version})")
```

Comment:
If `i` does not start with `prefix`, this will fail because `text` will not be defined